src/Controller.ts

```diff
@@ -38,9 +40,24 @@ export class Controller {
             debug ? await this._enableAllBPs() : await this._disableAllBPs();
 
             app.activeSessionName = targetConfig.name;
+            let jmxport = await(getPort());
+            app.jmxPort = jmxport;
+            let vmArgs = '-Dcom.sun.management.jmxremote ' +
+                `-Dcom.sun.management.jmxremote.port=${jmxport} `+ 
+                '-Dcom.sun.management.jmxremote.authenticate=false ' + 
+                '-Dcom.sun.management.jmxremote.ssl=false ' +
+                '-Djava.rmi.server.hostname=localhost '+ 
+                '-Dspring.application.admin.enabled=true';
+            if (targetConfig.vmArgs) {
+                //TODO: smarter merge? What if user is trying to enable jmx themselves on a specific port they choose, for example?
```

Comment:
This is a good question. I think later we can create an issue to discuss about it.